chat-plugins/info.js

```diff
@@ -1232,7 +1232,7 @@ exports.commands = {
 			buf += "<strong>" + (worker.pid || worker.process.pid) + "</strong> - Sockets " + i + "<br />";
 		}
 
-		const ProcessManager = require('./../process-manager');
+		const ProcessManager = require('../process-manager');
```

Comment:
I think the convention's `../process-manager`, but I see `./../file` pop up occasionally